lightning/src/chain/keysinterface.rs

```diff
@@ -274,11 +274,7 @@ pub trait ChannelKeys : Send+Clone {
 	/// htlc holds HTLC elements (hash, timelock) if the output being spent is a HTLC output, thus
 	/// changing the format of the witness script (which is committed to in the BIP 143
 	/// signatures).
-	///
-	/// on_remote_tx_csv is the relative lock-time that that our counterparty would have to set on
```

Comment:
Just to make sure I'm not missing anything, why was this not named `local_to_self_delay`?

Reply:
I believe the nomenclature was changed because local/remote and self is all super duper incredibly confusing...personally the remote_tx stuff is also confusing...I'm pushing for remote_broadcastable or broadcaster.

Reply:
Yes global naming is currently WIP, fyi https://github.com/rust-bitcoin/rust-lightning/pull/633. We should clearly dissociate transaction/signing context where data are labeled `local/remote` in function of whom is going to be the transaction owner from storage/caching context where data is labeled according to whom is issuer, either us or counterparty.

Reply:
OK, will keep an eye on this